strict.js

```diff
@@ -0,0 +1,25 @@
+
+ function  strictEqual() {
+     let plyrsreq = 52
+     if (FBteams == plyrsreq) {
+         return 'You have the amount of players required, lets play!'
+         
+     } else {
+         return 'sorry, you do not have enough players keep recruiting, try again next year!'
+         
+     }
+
+     
```

Comment:
now that we are working in node it would be a good idea to start making your js file index.js